chef-utils/README.md

```diff
@@ -0,0 +1,209 @@
+# Chef Utils gem
+
+**Umbrella Project**: [Chef Infra](https://github.com/chef/chef-oss-practices/blob/master/projects/chef-infra.md)
+
+**Project State**: [Active](https://github.com/chef/chef-oss-practices/blob/master/repo-management/repo-states.md#active)
+
+**Issues [Response Time Maximum](https://github.com/chef/chef-oss-practices/blob/master/repo-management/repo-states.md)**: 14 days
+
+**Pull Request [Response Time Maximum](https://github.com/chef/chef-oss-practices/blob/master/repo-management/repo-states.md)**: 14 days
+
+## Getting Started
+
+Chef Utils gem is common code and mixins for the core Chef Infra ruby gems.  This should be are "core library" or "foundations" library
+for the chef ecosystem (and external related gems) which allows the use of core code and utility functions of the chef gem without requiring
+all the heaviness of the chef gem.
+
+### Platform Helpers
+
+Individual platforms for when code MUST be different on a case-by-case basis.  It is generally encouraged to not use these and to use
+the `platform_family` helpers unless it is known that code must be special cased for individual platforms.
+
+* `aix_platform?`
+* `amazon_platform?`
+* `arch_platform?`
+* `centos_platform?`
+* `clearos_platform?`
+* `debian_platform?`
+* `dragonfly_platform?`
+* `fedora_platform?`
+* `freebsd_platform?`
+* `gentoo_platform?`
+* `leap_platform?`
+* `linuxmint_platform?`
+* `macos_platform?`
+* `netbsd_platform?`
+* `nexentacore_platform?`
+* `omnios_platform?`
+* `openbsd_platform?`
+* `openindiana_platform?`
+* `opensolaris_platform?`
+* `opensuse_platform?`
+* `oracle_platform?`
+* `raspbian_platform?`
+* `redhat_platform?`
+* `scientific_platform?`
+* `slackware_platform?`
+* `smartos_platform?`
+* `solaris2_platform?`
+* `suse_platform?`
+* `ubuntu_platform?`
+* `windows_platform?`
+
+For campatibility with old chef-sugar code the following aliases work for backwards compatibility, but will be DEPRECATED in the future.
+
+* `centos?`
+* `clearos?`
+* `linuxmint?`
+* `nexentacore?`
+* `omnios?`
+* `openindiana?`
+* `opensolaris?`
+* `opensuse?`
+* `oracle?`
+* `raspbian?`
+* `redhat?`
+* `scientific?`
+* `ubuntu?`
+
+### Platform Family Helpers
+
+These should be the most commonly used helpers to identify the type of node which group somewhat similar or nearly identical types of platforms.
+There are helpers here which are also meta-families which group together multiple types into supertypes.
+
+* `aix?`
+* `amazon?`
+* `arch?`
+* `debian?` - includes debian, ubuntu, linuxmint, raspbian, etc
+* `dragonflybsd?`
+* `fedora?`
+* `freebsd?`
+* `gentoo?`
+* `macos?`
+* `netbsd?`
+* `openbsd?`
+* `rhel?` - includes redhat, cnetos, scientific, oracle
+* `smartos?`
+* `solaris2?`
+* `suse?`
+* `windows?`
+* `windows_ruby_platform?` - this is always true if the ruby VM is running on a windows host and is not stubbed by chefspec
```

Comment:
i need to rename `windows_ruby_plaform?` i think i hate it now due to all the other `_platform` helpers.